modules/distribution/src/main/resources/log4j.properties

```diff
@@ -222,3 +222,35 @@ handlers=java.util.logging.ConsoleHandler
 
 # Replacing default INFO level with SEVERE
 java.util.logging.ConsoleHandler.level=SEVERE
+
+# moquette-log properties
+
+log4j.logger.org.dna.mqtt=INFO
+log4j.logger.org.dna.mqtt.moquette.messaging.spi.impl.ProtocolProcessor=INFO
+log4j.logger.org.dna.mqtt.moquette.messaging.spi.impl.SimpleMessaging=WARN
+
+#Protocol parsing
+log4j.logger.org.dna.mqtt.moquette.server.netty.NettyMQTTHandler=WARN
+#log4j.logger.org.dna.mqtt.moquette.server.netty.NettyMQTTHandler=DEBUG
+log4j.logger.org.dna.mqtt.moquette.parser.netty=WARN
+#Storage server
+#log4j.logger.org.dna.mqtt.moquette.messaging.spi.impl.subscriptions.SubscriptionsStore=DEBUG
+#log4j.logger.org.dna.mqtt.moquette.messaging.spi.impl.HawtDBStorageService=DEBUG
+
+# stdout appender is set to be a ConsoleAppender.
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+#log4j.appender.stdout.Threshold=WARN
+#log4j.appender.stdout.Threshold=INFO
+#log4j.appender.stdout.Threshold=DEBUG
+#log4j.appender.stdout.Threshold=TRACE
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%-4r [%t] %-5p %c{1} %x - %m%n
+
+#file appender
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.Threshold=INFO
+log4j.appender.file.File=moquette.log
```

Comment:
Correct me if I am wrong, are we logging into a different file? not wso2carbon.log ?